tools/SetupFlow/DevHome.SetupFlow/ViewModels/Environments/SelectEnvironmentProviderViewModel.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using CommunityToolkit.Mvvm.Messaging;
+using DevHome.Common.Contracts.Services;
+using DevHome.Common.Environments.Models;
+using DevHome.SetupFlow.Models.Environments;
+using DevHome.SetupFlow.Services;
+using Serilog;
+
+namespace DevHome.SetupFlow.ViewModels.Environments;
+
+public partial class SelectEnvironmentProviderViewModel : SetupPageViewModelBase
+{
+    private readonly ILogger _log = Log.ForContext("SourceContext", nameof(SelectEnvironmentProviderViewModel));
+
+    private readonly Microsoft.UI.Dispatching.DispatcherQueue _dispatcher;
```

Comment:
nit: using

Reply:
any reason why? wouldn't this be disposed when the class gets disposed?